rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1271,10 +1271,11 @@ private void processFundsMigration(Transaction rskTx) throws IOException {
                         availableUTXOs
                     );
                 } catch (Exception e) {
+                    // Funds cannot be migrated if there is no retiring federation
```

Comment:
This comment is a bit confusing, do we expect an exception here if there is no retiring federation